protocol/chainsync/client.go

```diff
@@ -570,6 +572,8 @@ func (c *Client) messageHandler(msg protocol.Message) error {
 		err = c.handleIntersectFound(msg)
 	case MessageTypeIntersectNotFound:
 		err = c.handleIntersectNotFound(msg)
+	case MessageTypeDone:
+		return nil
```

Comment:
This doesn't belong here. The client will never receive this message, only the server